pkgs/development/python-modules/falcon/default.nix

```diff
@@ -1,28 +1,70 @@
 { lib
 , buildPythonPackage
 , fetchPypi
+, pytestCheckHook
 , coverage
 , ddt
 , nose
 , pyyaml
 , requests
 , testtools
+, pytest-asyncio
+, aiofiles
+, cbor2
+, httpx
+, jsonschema
+, python_mimeparse
+, msgpack
+, mujson
+, orjson
+, pecan
+, rapidjson
+, ujson
+, websockets
+, gunicorn
+, hypercorn
+, uvicorn
 }:
 
 buildPythonPackage rec {
   pname = "falcon";
-  version = "2.0.0";
+  version = "3.0.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "eea593cf466b9c126ce667f6d30503624ef24459f118c75594a69353b6c3d5fc";
+    sha256 = "sha256-xB2E2zJYgahw6LcSnV7P2XL6QyPPd7cRmh0qIZZu5oE=";
   };
 
-  checkInputs = [coverage ddt nose pyyaml requests testtools];
+  checkInputs = [
+    pytestCheckHook
+    coverage
+    ddt
+    nose
+    pyyaml
+    requests
+    testtools
+    pytest-asyncio
+    aiofiles
+    cbor2
+    httpx
+    jsonschema
+    python_mimeparse
+    msgpack
+    mujson
+    orjson
+    pecan
+    rapidjson
+    ujson
+    websockets
+    gunicorn
+    hypercorn
+    uvicorn
+  ];
 
-  # The travis build fails since the migration from multiprocessing to threading for hosting the API under test.
-  # OSError: [Errno 98] Address already in use
-  doCheck = false;
+  disabledTestPaths = [
+    # nuts not packaged
```

Comment:
Can we package it instead?

Reply:
If it's only for tests, I would say it's not worth the constant maintenance burden of maintaining the package.